employees/common/exports.py

```diff
@@ -82,8 +82,11 @@ def generate_xlsx_for_project(self, project: Project) -> Workbook:
         self._workbook._sheets.sort(key=lambda w: str.lower(w.title))
         return self._workbook
 
-    def generate_xlsx_for_single_user(self, author: CustomUser) -> Workbook:
-        reports = author.get_reports_created().order_by("date", "project__name")
+    def generate_xlsx_for_single_user(self, author: CustomUser, project_id: int) -> Workbook:
+        if project_id is None:
```

Comment:
This can be simplified to:
```
reports = author.get_reports_created().order_by("date", "project__name")
if project_id is not None:
    reports = reports.filter(project=project_id)
```

Reply:
I will do this in next PR